exhibitor-appliance.yaml

```diff
@@ -19,6 +19,12 @@ SenzaInfo:
     - CustomS3BucketRegion:
         Description: "Use custom s3 bucket region for exhibitor configuration"
         Default: ""
+    - CustomTransactionsDirectory:
+        Description: "Use custom transactions directory"
+        Default: "/opt/exhibitor/transactions"
```

Comment:
If the `CustomTransactionsDirectory` and `CustomSnapshotsDirectory` are not specified in command line it will default to `/opt/exhibitor/transactions` and `/opt/exhibitor/snapshots` which are probably not writeable.
That means a new docker image wont work with this senza template.

Also be careful with i3 instances. AWS documentation tells that Ubuntu 14.04 (on which taupage is based) has outdated support of NVMe. New taupage will include a kernel from 16.04. But it's not released yet publicly.